src/Kentico.Xperience.Algolia.csproj

```diff
@@ -51,8 +51,8 @@
 
 	<ItemGroup>
 		<PackageReference Include="Algolia.Search" Version="6.13.0" />
-		<PackageReference Include="Kentico.Xperience.Admin" Version="22.3.0" />
-		<PackageReference Include="Kentico.Xperience.WebApp" Version="22.3.0" />
+		<PackageReference Include="Kentico.Xperience.Admin" Version="26.1.1" />
```

Comment:
Please increase the version number on line 15 to 2.0.0 as changing the dependency to new major version is considered as breaking change.